### ISSUE TITLE
[SUGESTION] Medic Module

### ISSUE THREAD

**Issue by pandathebeasty** [Ganter1234/Shop-Core#1]
**What should do?**
Heal the player when type !medkit or !medic (add amount of health to current health)

**Config parameters**
1. amount of health ("++health" will add amount of health to current but not more than maximum supported or "health" set health to this value)
2. max use per round
3. credit cost

**Others**
medkits should be stackable
medkits should decrease when are used

**What should be helpful**
command to buy an amount of items ( css_buy item_name amount[one if not specified] )

**Comment by Ganter1234** [comment]
I'll take into account that such a module is in the plans, thanks

**Comment by pandathebeasty** [comment]
hi, is this still on plan?

**Comment by Ganter1234** [comment]
> hi, is this still on plan?

Yes, we just need to improve the API a little to save the plugin from crutches
